Header header					            # ROS header
uint8 requester_idx				            # Identity number of the responding P4xx
uint8 responder_idx				            # Index number of the node in the responding array passed by rosparam
float32 distance				            # Distance measurement
